src/applications/gi/components/vet-tec/VetTecCalculator.jsx

```diff
@@ -11,6 +11,7 @@ import { getCalculatedBenefits } from '../../selectors/vetTecCalculator';
 import VetTecCalculatorForm from './VetTecCalculatorForm';
 import PropTypes from 'prop-types';
 import { ariaLabels } from '../../constants';
+import environment from '../../../../platform/utilities/environment';
```

Comment:
Need to use the platform alias path here

Reply:
updated